packages/components/pie-card-container/src/index.ts

```diff
@@ -40,12 +40,17 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
     @property({ type: Boolean })
     public isDraggable = false;
 
+    @property({ type: String, reflect: true })
```

Comment:
please add a TODO comment here to add the updated valid property values decorator and add the ticket number to the comment for reference :)

Reply:
Fixed.